Sources/MapboxSearch/InternalAPI/CoreSearchResultProtocol.swift

```diff
@@ -53,6 +53,9 @@ protocol CoreSearchResultProtocol {
     var distance: NSNumber? { get }
     var distanceToProximity: CLLocationDistance? { get }
 
+    /** map of external ids. */
```

Comment:
Could it be possible to use the DocC format of the comment?
```suggestion
    /// The map of external ids.
```